lib/cc/engine/analyzable_files.rb

```diff
@@ -19,7 +18,7 @@ def all
       def fetch_files(paths)
         paths.map do |path|
           if path =~ %r{/$}
-            Dir.glob("#{path}/**/*.coffee")
+            Dir.glob("#{path}**/*.coffee")
```

Comment:
This (and only this) sounds like the bug described in the PR description -- why the other changes? Anything we can break up into separate PRs?